sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -143,7 +150,7 @@
         ],
         "responses": {
           "200": {
-            "description": "Returns the list of Configuration objects",
+            "description": "Returns the list of Configuration objects. Pagination is not supported.",
```

Comment:
Why not ask for feature request?

Reply:
We are currently making SDK for existing IotHub features. The preview branch will have pnp stuff and anything new. We can surely ask them at that point. This change is mainly to send the service team content to update existing swagger comments. This will help up build without errors with amount of changes on our end. As for using the AsynPageable, we will not be able to use it if the service does not support. We can address the implementation part during API design. Note: This manual editing is not the approach we are taking to fix issues. This is just to send suggestions.